xbmc/utils/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(SOURCES ActorProtocol.cpp
             AlarmClock.cpp
             AliasShortcutUtils.cpp
+            AMLUtils.cpp
```

Comment:
We want cause of the aml_present - but if you know a way to only do that on Android, please tell me. New to cmake system.